src/commands/org/delete/scratch.ts

```diff
@@ -38,16 +38,18 @@ export default class DeleteScratch extends SfCommand<ScratchDeleteResponse> {
     const flags = (await this.parse(DeleteScratch)).flags;
     const resolvedUsername = flags['target-org'];
     const orgId = (await AuthInfo.create({ username: resolvedUsername })).getFields().orgId as string;
+    const typeScratch = (await AuthInfo.create({ username: resolvedUsername })).getFields().isScratch;
```

Comment:
nit:
you could do:
```ts
const { orgId, isScratch } = (await AuthInfo.create({ username: resolvedUsername })).getFields()
```

to avoid calling `AuthInfo.getFields` twice

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment